int64 mode
int64 lidar
int64 gamePad_x
int64 gamePad_z
int64 visual_x
int64 visual_z  


